### DISCUSSION TITLE
user level and xp progression

### DISCUSSION THREAD

**Discussion by blockchainrichs** [IBAX-io/go-ibax#454] [Ideas] [answered]
### Clear and concise description of the problem

I think it is insufficient to try to measure the point-based user contribution alone.

### Suggested solution

It would be great if there are levels and xp progress according to the contributions of the users.

### Alternative

interface and must be user levels and xp tiers that are really easy to follow and user friendly

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment, marked as answer]
> I think it is insufficient to try to measure the point-based user contribution alone.
> 
> ### Suggested solution
> It would be great if there are levels and xp progress according to the contributions of the users.
> 
> ### Alternative
> interface and must be user levels and xp tiers that are really easy to follow and user friendly


We have considered reputation and experience, and we will consider updating this features in the future.
